spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1641,14 +1522,14 @@ describe Crystal::Formatter do
   assert_format "foo = 1\n->foo.[](Int32)"
   assert_format "foo = 1\n->foo.[]=(Int32)"
 
-  assert_format "->{ x }"
-  assert_format "->{\nx\n}", "->{\n  x\n}"
-  assert_format "->do\nx\nend", "->do\n  x\nend"
-  assert_format "->( ){ x }", "->{ x }"
-  assert_format "->() do x end", "->do x end"
+  assert_format "-> { x }"
```

Comment:
**suggestion:** Keep the old formatting as input, and the new one as output. This has the added benefit of ensuring that the previous format transforms to the new style correctly.
`assert_format` checks idempotentcy of the output.

```suggestion
  assert_format "->{ x }", "-> { x }"
```

Ditto for all other spec changes.